composer.json

```diff
@@ -19,15 +19,15 @@
     "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ^2.6.6 || ~3.0",
-        "symfony/doctrine-bridge": "~2.3 || ~3.0",
+        "symfony/framework-bundle": "~2.7 || ~3.0",
+        "symfony/doctrine-bridge": "~2.7 || ~3.0",
```

Comment:
Symfony 2.3 is still supported.

AFAIK, @rande decided to keep the support of 2.3 on sonata projects for now.

Why not follow this?